java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsPermitsContentAccess.ql

```diff
@@ -0,0 +1,112 @@
+/**
+ * @name Android WebView settings permits content access
```

Comment:
We might leave this for the Docs team, but wouldn't `allows` be more appropriate (everywhere `permits` is used) due to how the relevant method is called?